src/commcare_cloud/commands/secrets.py

```diff
@@ -42,6 +44,10 @@ def _secrets_view(self, environment, secret_name):
     def _secrets_edit(self, environment, secret_name):
         environment.secrets_backend.prompt_user_input()
         secret_value = getpass.getpass("New value for '{}' secret '{}': ".format(environment.name, secret_name))
+        try:
+            secret_value = json.loads(secret_value)
```

Comment:
This converts the value _from_ json to a Python object (PR description says "convert secret _to_ json"). Should this be using `json.dumps`?

Reply:
No, this is correct. I've update the PR title